### DISCUSSION TITLE
Video Tutorial

### DISCUSSION THREAD

**Discussion by BillGunIO** [ModelSurge/sd-webui-comfyui#68] [Ideas]
I put together a quick tutorial on how to get this up and running. I figured I'd share it as a resource to the creators/users. Feel free to include in the official listing if you think it will help people.

[STABLE INCEPTION: Run ComfyUI in AUTOMATIC1111](https://www.youtube.com/watch?v=H5103u5uRII)

**Comment by ljleb** [comment]
I appreciate the time you spent making this content! Very cool.

There is 1 or 2 things I have to mention:

1. Copying usual webui resources (i.e. ckpt, vae, controlnet...) to the models directlory of comfyui should not be necessary anymore as of the latest version of the extension. If this does not work, please open an issue!
2. We are working on integrating comfyui in specific places in the workflow of the webui. For example, see https://github.com/ModelSurge/sd-webui-comfyui/pull/63 where @John-WL is working on getting comfyui to work as a custom preprocessor workflow
    - other mini-workflows will include custom processing of img2img init image and custom processing of controlnet input images among others

**Reply by BillGunIO** [reply]
Awesome. I'll keep an eye out for updates. If you want to let me know when you've added enough features to justify an updated video please let me know (billmeeks@everlyheights.tv) and I'll see if I can fit a video into my release schedule. Honestly thrilled with the extension so far, and excited that more integrations are coming!

**Reply by ljleb** [reply]
No problem! Thanks for the support, we'll get in touch when we consider the extension has become mature.
